setup_venv.sh

```diff
@@ -96,6 +96,7 @@ fi
 if [[ -z "${USE_IREE}" ]]; then
   RUNTIME="https://nod-ai.github.io/SHARK-Runtime/pip-release-links.html"
```

Comment:
You probably want to add a rm .use-iree here to handle the case of switching between one and another